install_gridpack.sh: cannot find -lBoost::mpi

When using the [install_gridpack.sh](https://github.com/GridOPTICS/GridPACK/blob/b17b1268542fd5c784a67936d4624ab1a188a866/install_gridpack.sh) script, these errors occur on various platforms when building the Python interface:
```
/usr/bin/ld: cannot find -lBoost::mpi: No such file or directory
/usr/bin/ld: cannot find -lBoost::serialization: No such file or directory
/usr/bin/ld: cannot find -lBoost::random: No such file or directory
/usr/bin/ld: cannot find -lBoost::filesystem: No such file or directory
/usr/bin/ld: cannot find -lBoost::system: No such file or directory
```

--- a/install_gridpack.sh
+++ b/install_gridpack.sh
@@ -55,6 +55,8 @@ then
    -D Boost_DIR:STRING=${GP_EXT_DEPS}/boost_1_78_0/install_for_gridpack/lib/cmake/Boost-1.78.0 \
    -D Boost_LIBRARIES:STRING=${GP_EXT_DEPS}/boost_1_78_0/install_for_gridpack/lib \   
    -D Boost_INCLUDE_DIRS:STRING=${GP_EXT_DEPS}/boost_1_78_0/install_for_gridpack/include \
+   -D Boost_NO_BOOST_CMAKE:BOOL=TRUE \
+   -D Boost_NO_SYSTEM_PATHS:BOOL=TRUE \
    -D PETSC_DIR:PATH=${GP_EXT_DEPS}/petsc/install_for_gridpack \                  
    -D MPI_CXX_COMPILER:STRING='mpicxx' \     
    -D MPI_C_COMPILER:STRING='mpicc' \                                         

